### PR TITLE
docs: add docs for property `fluentd.logs.containers.sourceHost`

### LINKED ISSUE #1782: Remove `.Values.fluentd.logs.containers.sourceHost` from the chart
`.Values.fluentd.logs.containers.sourceHost` is already not referenced anywhere in `values.yaml` so let's remove it.

This would be a breaking change since users had the possibility to manually add this to `values.yaml` and observe the effect of that value changed.

https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/d87aab8d99d63c3ff364c72b85aa9cd2cba4855f/deploy/helm/sumologic/conf/logs/fluentd/logs.kubernetes.sumologic.filter.conf#L2

### PR DESCRIPTION
**Pull request by astencel-sumo**
The property has always existed  and is used in `logs.kubernetes.sumologic.filter.conf` file, but was missing from `values.yaml` and docs.

Fixes #1782

### CODE REVIEW

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -1971,6 +1973,18 @@ kube-prometheus-stack:
...
+          writeRelabelConfigs:
+            - action: keep
+              regex: kube-state-metrics;(?:kube_statefulset_status_observed_generation|kube_statefulset_status_replicas|kube_statefulset_replicas|kube_statefulset_metadata_generation|kube_daemonset_status_current_number_scheduled|kube_daemonset_status_desired_number_scheduled|kube_daemonset_status_number_misscheduled|kube_daemonset_status_number_unavailable|kube_deployment_spec_replicas|kube_deployment_status_replicas_available|kube_deployment_status_replicas_unavailable|kube_node_info|kube_node_status_allocatable|kube_node_status_capacity|kube_node_status_condition|kube_hpa_spec_max_replicas|kube_hpa_spec_min_replicas|kube_hpa_status_(condition|(current|desired)_replicas))
+              sourceLabels: [job, __name__]
+            - action: labelmap
+              regex: (pod|service)
+              replacement: service_discovery_${1}
+            - action: labeldrop
+              regex: (pod|service|container)
+        ## kube service metrics
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/values.yaml:1987]: ?

File: deploy/helm/sumologic/values.yaml [hunk 2]
```diff
@@ -3936,6 +3950,13 @@ metadata:
           attributes:
             - action: delete
               key: k8s.pod.pod_name
+        resource/normalize_ot_labels:
+          attributes:
+            - action: upsert
+              from_attribute: k8s_service_name
+              key: k8s.service.name
+            - action: delete
+              key: k8s_service_name
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/values.yaml:3959]: ?

File: deploy/helm/sumologic/values.yaml (hunk 1 above)
Comment (astencel-sumo) [reply deploy/helm/sumologic/values.yaml]: Fixed, thanks @sumo-drosiek.

File: deploy/helm/sumologic/values.yaml (hunk 2 above)
Comment (astencel-sumo) [reply deploy/helm/sumologic/values.yaml]: Sorry, fixed.

**Review by sumo-drosiek — APPROVED** [review]
:drum:

**Pull request merged**